src/d3a/d3a_core/util.py

```diff
@@ -336,6 +359,12 @@ def recursive_retry(functor, retry_count, max_retries, *args, **kwargs):
         return recursive_retry(functor, retry_count+1, max_retries, *args, **kwargs)
 
 
+def change_global_config(**kwargs):
+    for arg, value in kwargs.items():
+        if hasattr(GlobalConfig, arg):
```

Comment:
I would either raise an exception or assert in the else block here, because if I understand correctly only if the user tries to monkeypatch the GlobalConfig object with additional attributes.

Reply:
I think it is not possible for the user to monkeypatch the GlobalConfig object, but I will add the else block anyway to be on the save side and to be more explicit. Thanks!

Reply:
Ahh no, this else block has to be empty, as I like to do this: `change_global_config(**configuration.__dict__)` 
but configuration.__dict__ might not only contain members of GlobalConfig.
Does this make sense?

Reply:
You can filter out the class dict items like this : 
config_dict = {k: v for k, v in vars(GlobalConfig).items() if not k.startswith("__")}
but I am not sure if this is an overkill for us here.

Reply:
Thanks. I tried to implement it, but i came to the conclusion, that it is a bit of an overkill.
I you dont mind @spyrostz I will keep the else block empty. Or shall I use `pass` and add a comment why this block is empty?